src/Geoscape/GeoscapeState.h

```diff
@@ -69,8 +70,11 @@ class GeoscapeState : public State
 	int _slowdownCounter;
 
 	// hidden alien activity accumulators
-	std::map<OpenXcom::Region*, int> _hiddenAlienActivityRegions;
-	std::map<OpenXcom::Country*, int> _hiddenAlienActivityCountries;
+	std::unordered_map<entt::entity, int> _hiddenAlienActivityRegions;
+	std::unordered_map<entt::entity, int> _hiddenAlienActivityCountries;
+
+	std::unordered_map<size_t, entt::entity> _countryDebugComboBoxIndexToEntity;
```

Comment:
are these debug maps needed still?

Reply:
Lol I would love to nuke them. This section was such a pain. But presumably modders need/want them when setting up custom regions so the feature has value.